multiqc/utils/search_patterns.yaml

```diff
@@ -230,7 +228,6 @@ clipandmerge:
   num_lines: 5
 clusterflow/logs:
   fn: "*_clusterFlow.txt"
-  shared: true
```

Comment:
This is maybe the only one that should be left there. ClusterFlow is my old workflow manager tool and it concatenates log files. So if there are any ClusterFlow logs then they definitely will be shared.